tests/media/client/ipc/mediaPipelineIpc/GetVolumeTest.cpp

```diff
@@ -0,0 +1,102 @@
+/*
+ * If not stated otherwise in this file or this component's LICENSE file the
+ * following copyright and licenses apply:
+ *
+ * Copyright 2022 Sky UK
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "MediaPipelineIpcTestBase.h"
+
+MATCHER_P(GetVolumeRequestMatcher, sessionId, "")
+{
+    const ::firebolt::rialto::GetVolumeRequest *request = dynamic_cast<const ::firebolt::rialto::GetVolumeRequest *>(arg);
+    return ((request->session_id() == sessionId));
+}
+
+class RialtoClientMediaPipelineIpcGetVolumeTest : public MediaPipelineIpcTestBase
+{
+protected:
+    virtual void SetUp()
+    {
+        MediaPipelineIpcTestBase::SetUp();
+
+        createMediaPipelineIpc();
+    }
+
+    virtual void TearDown()
+    {
+        destroyMediaPipelineIpc();
+
+        MediaPipelineIpcTestBase::TearDown();
+    }
+};
+
+/**
+ * Test that getVolume can be called successfully.
+ */
+TEST_F(RialtoClientMediaPipelineIpcGetVolumeTest, Success)
+{
+    expectIpcApiCallSuccess();
+
+    EXPECT_CALL(*m_channelMock, CallMethod(methodMatcher("getVolume"), m_controllerMock.get(),
+                                           GetVolumeRequestMatcher(m_sessionId), _, m_blockingClosureMock.get()));
+
+    double volume;
+    EXPECT_TRUE(m_mediaPipelineIpc->getVolume(volume));
```

Comment:
Set response and check volume param set correctly.

Reply:
done